vissl/data/ssl_dataset.py

```diff
@@ -118,6 +118,16 @@ def _get_data_limit_sampling(cfg: AttrDict, split: str) -> AttrDict:
         )
         return cfg["DATA"][split].get("DATA_LIMIT_SAMPLING", default_sampling)
 
+    def set_epoch(self, epoch):
```

Comment:
hmm this becomes a little confusing since it's special case for the Airstore dataset only. We should add:
1) a clear comment to this function explaining why we set_epoch for airstore and why this isn't require for non-airstore datasets 
2) assert to check the data is airstore

Reply:
recent change directly checks dataset is an instance of AirstoreDataSet, so set_epoch and set_start_iter don't need be added to GenericSSLDataset any more